zpm.c

```diff
@@ -399,6 +399,27 @@ int plugin_print_list() {
     return 0;
 }
 
+int plugin_print_script() {
+    char entry[PATH_MAX];
+    char* zpm_init = get_plugin_list_path();
+    FILE* store = fopen(zpm_init, "r");
+
+    if (!store) {
+        // migth need an error message here.
```

Comment:
printf("Could not open \"%s\". Check the file exists and can be read.", zpm_init);